Article 119: Supervised Learning

Garcko opened the pull request:
## Attributes

<!-- Please use one of these items per category by removing the comment tag at the beginning and end of the line. -->

#### Length of text ####
:page_with_curl: short 
<!-- :book:  long -->

####  Difficulty to read ####
 :mouse:  light 
<!-- :elephant:  heavy -->

####  Difficulty to understand #### 
 :school_satchel:  easy 
<!-- :mortar_board:  difficult -->

####  Type of text #### 
<!-- :laughing:  funny -->
 :bowtie:  informative 
<!-- :sunglasses:  catchy -->

####  Type of header #### 
<!-- :santa:  funny --> 
<!-- :bulb:  informative -->
 :boom:  catchy

s3lph commented on articles/article-119-supervised-learning.md:
```diff
@@ -0,0 +1,36 @@
+
+<!-- BEGIN OPTIONAL -->
+## Remark
+* Remove the overfitting part.
```
Please don't put review notes into the optional text section. Instead, add them as a comment to the pull request.

Fjolyo commented:
# Plenum Suggestions

- Mention problem instead of example

- Title is not explained enough / confusing

- Maybe mention supervised/unsupervised

- Don't mention "overfitting"

MoiraZuber commented:
Added an image suggestion.

Garcko commented:
changed the title, removed remark and figure text,  removed the overfitting part,  I am not sure what to change for "mention problem instead of example"

Fjolyo commented:
We think the content of the article is good, but the sentences and the grammar needs some adjustments.
Maybe the grammar team can help here.
